staging/src/k8s.io/apiextensions-apiserver/test/integration/validation_test.go

```diff
@@ -777,8 +778,8 @@ func TestCRValidationOnCRDUpdate(t *testing.T) {
 			}
 
 			// CR is now accepted
-			err = wait.Poll(500*time.Millisecond, wait.ForeverTestTimeout, func() (bool, error) {
-				_, err := noxuResourceClient.Create(context.TODO(), instanceToCreate, metav1.CreateOptions{})
+			err = wait.PollUntilContextTimeout(context.Background(), 500*time.Millisecond, wait.ForeverTestTimeout, true, func(ctx context.Context) (done bool, err error) {
+				_, err = noxuResourceClient.Create(ctx, instanceToCreate, metav1.CreateOptions{})
```

Comment:
Please leave the `:=`.

Reply:
Fixed it https://github.com/kubernetes/kubernetes/commit/ed7f7ce92b06b2452a3d879b4c70d34f5d31ff23